php/libraries/LorisForm.class.inc

```diff
@@ -1180,7 +1182,7 @@ class LorisForm
      */
     function isSubmitted()
     {
-        return (!empty($_FILES) || !empty($_POST));
+        return (!empty($_FILES) || !empty($_POST) || $this->directEntry);
```

Comment:
Since the new way of submitting Survey fields is with PUT and PATCH requests this would evaluate to false causing the module to not work. Add a flag for direct entry to bypass the check.